blogview/templates/all_posts.html

```diff
@@ -30,13 +30,23 @@
                     </div>
                   </div>
                 </div>
+                <audio id = "{{ post.music_link }}" controls style="display: none;">
```

Comment:
@aashutoshrathi You are creating one audio tag for each post. Don't you think it's more data consuming.

Reply:
Any other suggestions then @mohitkyadav ?

Reply:
> Why not create a single audio and change the src on mouseenter?

Reply:
Let's create audio in base.

Reply:
I think it is for the best, we already had an audio tag in each post. :confused: